src/build.d

```diff
@@ -520,7 +524,11 @@ alias runCxxHeadersTest = makeRule!((builder, rule) {
                         ") doesn't match with the reference header file (" ~
                         cxxHeaderReferencePath ~ ")\n";
                     auto diff = tryRun(["git", "diff", "--no-index", cxxHeaderReferencePath, cxxHeaderGeneratedPath], runDir).output;
-                    diff ~= "\n===============\nNOTE: You can rerun with AUTO_UPDATE=1 to automatically update the reference header file.";
+                    diff ~= "\n===============
+Detected changes to the frontend's C++ interface. Please ensure that those changes are
+reflected in the manually managed header files (check `<file>.h` when changing `<file>.d`).
+
+NOTE: Rerun with AUTO_UPDATE=1 to automatically update the reference header file.";
```

Comment:
```suggestion
The file `src/dmd/frontend.d` seems to be out of sync.
This is likely because changes were made which affect the C++ interface used by GDC and LDC.
Make sure that those changes have been properly reflected in the relevant header files (e.g. `src/dmd/scope.h` for changes in `src/dmd/dscope.d` or `src/dmd/statement.h` for changes in `src/dmd/statement.d`).

To update `frontend.h` and fix this error, run `./src/build.d cxx-headers-test AUTO_UPDATE=1`.
Note that the generated code need not be valid, as `frontend.h` is still under development.

To read more about `frontend.h` and its usage, see https://github.com/dlang/dmd/blob/master/src/README.md#cxx-headers-test
```

The most important IMO is to provide a command to copy paste. I hardcoded `./src/build.d` but we  should probably use `args[0]`.

Reply:
Done but I've slightly rephrased/reformatted the text to make it more concise.

> we should probably use args[0].

Done but it's kinda ugly when using `rdmd` / `dmd -run`:

> ```
> To update `frontend.h` and fix this error, run the following command:
> 
> `/tmp/dmd_runmklsoW cxx-headers-test AUTO_UPDATE=1`
> ```